README.rst

```diff
@@ -126,6 +126,9 @@ for ev3dev. The first allows you to run this program from Brickman, while the
 second imports this library.
 
 When saving Python files, it is best to use the ``.py`` extension, e.g. ``my-file.py``.
+Your program must be executable. To mark a program as executable run ``chmod +x my-file.py``
```

Comment:
Can you add a lead-in sentence/phrase here to make it clear that this is all part of the same idea? e.g. "To be able to run your Python code, your program must be executable."